tests/smoke/events.js

```diff
@@ -145,7 +145,8 @@
 				i,
 				root;
 
-			for (i=0, event=events[i]; i<events.length; i++) {
+			for (i=0; i<events.length; i++) {
```

Comment:
doh.  pretty sure I initially was trying to be clever and just ended up just not iterating.  Good catch that makes sense now.